kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/csv/CsvBuildStageExecution.scala

```diff
@@ -0,0 +1,24 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.csv
+
+/* Due to interoperability issues between Groovy and Scala
+ * this has to be declared as a mutable class with a default
+ * constructor. We can't use a case class.
```

Comment:
Also you may want to consider http://www.scala-lang.org/api/2.12.x/scala/beans/BeanProperty.html if this related in any way to JavaBeans